web/docs/auth/ui.md

```diff
@@ -1,3 +1,622 @@
 ---
 title: Auth UI
 ---
+
+import { EmailPill, UsernameAndPasswordPill, GithubPill, GooglePill } from "./Pills";
+
+To make using authentication in your app as easy as possible, Wasp generates the server-side code but also the client-side UI for you. It enables you to quickly get the login, signup, password reset and email verification flows in your app.
+
+Below we cover all of the available UI components and how to use them.
+
+![Auth UI](/img/authui/all_screens.gif)
+
+## How Auth UI works in Wasp?
+
+After Wasp generates the UI components for your auth, you can use it as is, or customize it to your liking.
+
+Based on your `main.wasp` file on the authentication providers you enabled, the Auth UI will show the corresponding UI (form and buttons). 
+
+For example, if you enabled e-mail authentication:
+
+<Tabs groupId="js-ts">
+<TabItem value="js" label="JavaScript">
+
+```wasp {5} title="main.wasp"
+app MyApp {
+  //...
+  auth: {
+    methods: {
+      email: {},
+    },
+    // ...
+  }
+}
+```
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```wasp {5} title="main.wasp"
+app MyApp {
+  //...
+  auth: {
+    methods: {
+      email: {},
+    },
+    // ...
+  }
+}
+```
+</TabItem>
+</Tabs>
+
+You'll get the following UI:
+
+![Auth UI](/img/authui/login.png)
+
+And then if you enable Google and Github:
+
+<Tabs groupId="js-ts">
+<TabItem value="js" label="JavaScript">
+
+```wasp title="main.wasp" {6-7}
+app MyApp {
+  //...
+  auth: {
+    methods: {
+      email: {},
+      google: {},
+      github: {},
+    },
+    // ...
+  }
+}
+```
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```wasp title="main.wasp" {6-7}
+app MyApp {
+  //...
+  auth: {
+    methods: {
+      email: {},
+      google: {},
+      github: {},
+    },
+    // ...
+  }
+}
+```
+</TabItem>
+</Tabs>
+
+The form will automatically update to look like this:
+
+![Auth UI](/img/authui/multiple_providers.png)
+
+Let's go though all of the available components and how to use them.
+
+## Auth components
+
+The following components are available for you to use in your app:
+1. [Login form](#1-login-form)
+2. [Signup form](#2-signup-form)
+3. [Forgot password form](#3-forgot-password-form)
+4. [Reset password form](#4-reset-password-form)
+5. [Verify email form](#5-verify-email-form)
```

Comment:
I think this shouldn't be an ordered list, and the sections for each form shouldn't have numbers either.